### PR TITLE
chore: updating readme

### PR DESCRIPTION
**Pull request by jpvajda**
This update provides better instructions on using the catalog manager to update applications in the catalog.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,22 @@ This project is used by New Relic to manage the collection of open source Nerdpa
 
 Currently, this can be used to view and _initiate_ updates to the Catalog (via a set of GitHub Actions workflows) by generating new Pull Requests to the [nr1-catalog](https://github.com/newrelic/nr1-catalog). You can also view existing Pull Requests to `nr1-catalog`, as well as see the state of Workflows.
 
+## First Time Additions to the Catalog
+
+If you are adding an application to the catalog for the first time you will need to change the application subscription model to `GLOBAL`. Therefore you will need to make a request to `#help-nr1-platformservices` via Slack to change the model of the application.
```
Comment (zstix) [inline README.md]: What if the contributor isn't a relic?

Comment (jpvajda) [reply README.md]: @zstix they way I understand this process is that the request to change an application from `owner_and_allowed` (3rd party custom apps) to `Global` (3rd party global apps) has to be done by a Relic, so the contributor would submit a request to do so via the repo: pathpoint is an example: https://github.com/newrelic/nr1-pathpoint where recently made this app global.

Comment (jpvajda) [reply README.md]: I made clarified this section, hopefully that helps.

Comment (zstix) [reply README.md]: I guess my point is: should we be documenting something in a public repository thats only intended for internal purposes? I'll approve the PR and let you decide.

**Review by zstix — APPROVED** [review]

**Pull request merged**